Updates scan interval notice of Toon component (#8975)
home-assistant/home-assistant.github.io: source/_components/toon.markdown

It also supports setting the temperature manually.

> The Toon API is polled at a 300-second interval, so the status is relatively fresh without overloading the API.

Reviewer: I would personally not cal 5 minutes fresh. But whatever you decide.
```suggestion
The Toon API is polled at a 300-second interval, so the API is not overloded.
```
